ospfd/ospf_vty.c

```diff
@@ -8925,6 +8925,31 @@ DEFUN (no_ospf_max_metric_router_lsa_shutdown,
 	return CMD_SUCCESS;
 }
 
+DEFUN (ospf_proactive_arp,
+       ospf_proactive_arp_cmd,
+       "proactive-arp",
+       "Allow sending ARP requests proactively\n")
+{
+	VTY_DECLVAR_INSTANCE_CONTEXT(ospf, ospf);
+
+	ospf->proactive_arp = 1;
```

Comment:
Nit: how about using `bool` and `true`/`false`? It would also match the YANG.

Reply:
Alright, changed